build.gradle

```diff
@@ -243,6 +243,9 @@ project('reactor-extra') {
   dependencies {
 	optional "org.eclipse.swt:org.eclipse.swt.${getPlatform()}:${swtVersionPlatform}"
 	optional("org.jetbrains.kotlin:kotlin-stdlib:${kotlinVersion}")
+	optional 'org.openjfx:javafx-base:12-ea+7'
```

Comment:
you probably want to use `getPlatform` like for the swt one, and avoid using an early access version...